src/web/components/SlotBodyEnd/ReaderRevenueEpic.tsx

```diff
@@ -162,37 +167,37 @@ export const canShow = ({
 		isPaidContent,
 		tags,
 		contributionsServiceUrl,
-	})
-		.then((contributionsPayload) =>
-			getBodyEnd(
-				contributionsPayload,
-				`${contributionsServiceUrl}/epic${queryString}`,
-			),
-		)
-		.then((response) => {
-			dataPerf.end();
-			return checkForErrors(response);
-		})
-		.then((response) => response.json())
-		.then((json: { data?: PreEpicConfig }) => {
-			if (!json.data) {
-				return { result: false };
-			}
-
-			const { meta, module } = json.data;
-
-			return {
-				result: true,
-				meta: {
-					meta,
-					module,
-					onReminderOpen: sendOphanReminderOpenEvent,
-				},
-			};
-		});
+		idApiUrl,
+	});
+
+	const response = await getBodyEnd(
+		contributionsPayload,
+		`${contributionsServiceUrl}/epic${queryString}`,
+	);
+
+	checkForErrors(response);
```

Comment:
`checkForErrors` throws if it's not a 200. Elsewhere in this `canShow` we return a `Promise.resolve(...)`, and also simply an object.
I'm sure it all works fine, but it would be nice to make this consistent thoughout the function.

Reply:
discussed IRL - it's a little confusing, but okay for now!